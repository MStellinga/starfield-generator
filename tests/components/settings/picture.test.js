/* eslint-disable jest/valid-expect */
import { h } from 'preact';
import render from 'preact-render-to-string';
import Picture from '../../../src/components/settings/picture';

describe('Test Image settings', () => {
	test('Image settings renders', () => {
        const tree = render(<Picture width={1} height={2} cluster={[]} />);
        expect(tree).toMatchSnapshot();	
    });

    // Unfortunately, adding tests that call event handlers is hard to do with Enzyme and function-based Preact components (at this time, anyway)
});
