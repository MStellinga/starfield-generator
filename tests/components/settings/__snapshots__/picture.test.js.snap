// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Image settings Image settings renders 1`] = `"<div class=\\"picture\\" style=\\"width: 1px;\\"><div class=\\"pictureEditButtons\\"><button>Add</button><button>Move</button></div><canvas width=\\"1\\" height=\\"2\\" class=\\"starcanvas\\"></canvas></div>"`;
